src/components/ebay-textbox/index.js

```diff
@@ -0,0 +1,31 @@
+const markoWidgets = require('marko-widgets');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const classes = ['textbox__control'];
+    if (input.fluid) {
+        classes.push('textbox__control--fluid');
+    }
+    return {
+        classes,
+        htmlAttributes: processHtmlAttributes(input),
+        rootClass: input.class || '',
+        value: input.value,
+        tag: input.fluid ? 'div' : 'span',
+        textBoxTag: Boolean(input.multiline) ? 'textarea' : 'input',
+        disabled: Boolean(input.disabled),
+        placeholder: input.placeholder,
```

Comment:
You don't need to process `placeholder` here or in the template. This will get handled through `htmlAttributes`.